src/main/java/net/sf/jabref/JabRefGUI.java

```diff
@@ -88,7 +88,6 @@ public static void checkForNewVersion(boolean manualExecution) {
 
     private void openWindow() {
         // Perform checks and changes for users with a preference set from an older JabRef version.
-        PreferencesMigrations.replaceAbstractField();
```

Comment:
Because the code checks if GENERAL_FIELDS contains `abstract` and in that case removes if from `GENERAL_FIELDS`.

The thing is that the only place `GENERAL_FIELDS` is used in the current master is for setting a default value...

``` Java
// The general fields stuff is made obsolete by the CUSTOM_TAB_... entries.
        defaults.put(GENERAL_FIELDS, "crossref;keywords;file;doi;url;urldate;"
                + "pdf;comment;owner");
```